doc/howto/instances_troubleshoot.md

```diff
@@ -59,7 +59,41 @@ To troubleshoot the problem, complete the following steps:
    If it is, and if you cannot figure out the source of the error from the log information, open a question in the [forum](https://discourse.ubuntu.com/c/lxd/).
    Make sure to include the log files you collected.
 
-## Troubleshooting example
+## Troubleshooting examples
+
+### Debug `systemd` `init`
+
+Here is how to ask `systemd` to enable [debug level messages](https://www.freedesktop.org/wiki/Software/systemd/Debugging/):
+
+```{terminal}
+:input: lxc config set systemd raw.lxc 'lxc.init.cmd = /sbin/init systemd.log_level=debug'
+
+:input: lxc start systemd
+```
```

Comment:
We're only showing the input here, so it's sufficient to use a normal code block (enclosed with ```` ``` ````).

The idea of the terminal view is to distinguish between input and output, and we don't need that here.